client/src/document/organisms/toc/index.scss

```diff
@@ -57,3 +57,57 @@
     display: block;
   }
 }
+
+.place {
```

Comment:
Could this go in a separate `placement.scss` and rather be imported here?

Reply:
ok